engine/commands/assign_reviewer_command.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 Explore.dev Unipessoal Lda. All Rights Reserved.
+// Use of this source code is governed by a license that can be
+// found in the LICENSE file.
+
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var (
+	assignReviewerCommandRegex = regexp.MustCompile(`^\/reviewpad assign-reviewers\s+((?:[a-zA-Z0-9\-]+[a-zA-Z0-9])(?:,\s*[a-zA-Z0-9\-]+[a-zA-Z0-9])*)(?:\s+(\d+))?(?:\s+(random|round-robin|reviewpad))?$`)
+)
+
+func AssignReviewerCommand(matches []string) ([]string, error) {
```

Comment:
Since this is inserted in the commands package we know that we are dealing with `assign reviewers` in a command context so I believe there is no need to mention `Command`.
```suggestion
func AssignReviewer(matches []string) ([]string, error) {
```

The same applies to `assignReviewerCommandRegex` that can be `assignReviewerRegex`. And also the file names, which can be `assignReviewer.go` and `assignReviewer_test.go` instead of `assign_reviewer_command.go` and `assign_reviewer_command_test.go`, respectively.